File: src/API/src/Modules/Clients/Clients.Application/ViewModels/ClientLookupBundleViewModel.cs
namespace Clients.Application.ViewModels
{
    public class ClientLookupBundleViewModel
    {
        public List<LookupViewModel> ClientTypes { get; set; } = new();
        public List<LookupViewModel> Countries { get; set; } = new();
        public List<LookupViewModel> Industries { get; set; } = new();
        public List<ClientUserLookupViewModel> AccountOwners { get; set; } = new();
    }
}

File: src/ClientApp/src/app/pages/clients/view-models/client-lookup-bundle.view-model.ts
import { LookupViewModel } from '../../leads/view-models/lookup.view-model';
import { ClientUserLookupViewModel } from './client-user-lookup.view-model';

export interface ClientLookupBundleViewModel {
    clientTypes: LookupViewModel[];
    countries: LookupViewModel[];
    industries: LookupViewModel[];
    accountOwners: ClientUserLookupViewModel[];
}
